generic/system/metadata.yaml

```diff
@@ -46,28 +46,28 @@ policyRules:
     - name: MITRE-TTP-T1082
       url:
       - https://attack.mitre.org/techniques/T1082/
-    tldr: System Information Discovery - block system owner discovery commands
+    tldr: Limit adversaries from gathering system information
     detailed: An adversary may attempt to get detailed information about the operating system and hardware, including
       version, patches, hotfixes, service packs, and architecture. Adversaries may use the information from System
       Information Discovery during automated discovery to shape follow-on behaviors, including whether or not the
       adversary fully infects the target and/or attempts specific actions.
   yaml: ksp-mitre-system-owner-user-discovery.yaml
-- name: write-under-bin-dir
-  precondition: 
-  - /bin/*
-  - OPTSCAN
-  description:
-    refs:
-    - name: NIST-SI-4
-      url:
-      - https://csf.tools/reference/nist-sp-800-53/r5/si/si-4/
-    tldr: System and Information Integrity - System Monitoring make directory under /bin/
-    detailed: System monitoring includes external and internal monitoring. External monitoring
-      includes the observation of events occurring at system boundaries. Internal monitoring
-      includes the observation of events occurring within the system. Organizations monitor systems,
-      for example, by observing audit activities in real time or by observing other system aspects 
-      such as access patterns, characteristics of access, and other actions.
-  yaml: ksp-nist-si-4-mkdir-bin-dir.yaml
+#- name: write-under-bin-dir
+#  precondition: 
+#  - /bin/*
+#  - OPTSCAN
+#  description:
+#    refs:
+#    - name: NIST-SI-4
+#      url:
+#      - https://csf.tools/reference/nist-sp-800-53/r5/si/si-4/
+#    tldr: System and Information Integrity - System Monitoring make directory under /bin/
+#    detailed: System monitoring includes external and internal monitoring. External monitoring
+#      includes the observation of events occurring at system boundaries. Internal monitoring
+#      includes the observation of events occurring within the system. Organizations monitor systems,
+#      for example, by observing audit activities in real time or by observing other system aspects 
+#      such as access patterns, characteristics of access, and other actions.
+#  yaml: ksp-nist-si-4-mkdir-bin-dir.yaml
```

Comment:
why are we removing this? if its duplicate can we delete it instead of commenting it out?
```suggestion

```

Reply:
Yes, it is a duplicate policy. The same rules are present in the file integrity policy. I noticed other policies commented out in the metadata, so I followed the same commenting procedure.